src/lib/fetchTickerData.ts

```diff
@@ -0,0 +1,47 @@
+import fetch, { Response } from 'node-fetch';
```

Comment:
this file is very similar to https://github.com/guardian/contributions-service/blob/master/src/lib/fetchTickerData.ts

Reply:
I do not think using `node-fetch` here is a good idea, as this lib is called in the frontend part of the app. I would favor using the native `fetch` API instead.

Note: `reader-revenue-links` is hydrated client side so you can use JS without having to worry about SSR

Reply:
William is right here, we don't need node-fetch, can we please refactor this out?

This should be using the lib/api functionality I think

Reply:
See https://github.com/guardian/dotcom-rendering/pull/1690

Reply:
🤟